Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:alpine as builder
+
+WORKDIR /app
+
+COPY . .
```

Comment:
tip: each command in dockerfile generates distinct layer. The subsequent builds are reusing these layers as long as their inputs are identical. Every time something changes in cwd everything after line 5 has to be regenerated. If you move the `COPY . .` after `RUN update-ca-certificates` the second build onwards will be much faster since the layers with the certificates can be simply reused.

Reply:
Thanks! Updated